docs/content/en/backends/ssh.md

```diff
@@ -5,52 +5,113 @@ weight: 2
 
 Use remote filesystem through SSH as the repository backend.
 
-## Configuration
-To use SSH backend, you have to configure `~/.ssh/config`.  
+## Features
 
-1. Check `ssh` command is available
+- Use the original `~/.ssh/config` without pain
+- Support password authentication
+- Support public key authentication
+- Support SSH agent
+- Support SSH proxy by `ProxyCommand` or `ProxyJump`
+- Support host key checking through `~/.ssh/known_hosts`
+- Concurrent uploading and downloading
 
-   ```
-   ssh -V
-   ```
+## Configuration
+To use SSH backend, you have to configure `~/.ssh/config`.  
 
 1. Configure the `~/.ssh/config`
     ```
-    # ~/.ssh/config
-    Host ml-vm
-        HostName <vm address>
+    Host myserver
+        HostName myserver.hosts
         User myname
-        IdentityFile ~/.ssh/id_myname
-
-    Host ml-vm2
-        HostName <vm address>
-        User myname
-        Port 2222
-        IdentityFile ~/.ssh/id_myname
+        IdentityFile ~/.ssh/id_ed25519
     ```
 
-1. Check if you can access the remote without password
+1. Check if you can access the SSH server successfully
 
     ```
-    ssh mydataset
+    ssh myserver
     ```
 
-For more information, please see [ssh config manual](https://linux.die.net/man/5/ssh_config)
+For more information, please see [ssh config manual](https://man.openbsd.org/ssh_config)
 
 ## Usage
 
 Init a workspace
 
 ```shell
-# avc init ssh://<host>/<path>
-avc init ssh://ml-vm/home/myname/datasets
+avc init myserver:path/to/mydataset
 ```
 
 Clone a repository
 
 ```shell
-avc clone ssh://ml-vm/home/myname/mydataset
+avc clone myserver:path/to/mydataset
 cd mydataset/
 ```
 
-> Note: the path has to be an absolute path. It is not a relative path to your home directory.
\ No newline at end of file
+## SSH Proxy
+
+There are two ways to connect to the destination server through bastion (proxy) server.
+
+### ProxyCommand
+
+1. Configure `~/.ssh/config`
+    ```
+    host bastion
+        hostname mybastion.hosts
+        user myname
+        identityfile ~/id_ed25519
+
+    host myserver
+        hostname myserver.hosts
+        user myname
+        identityfile ~/id_ed25519
+        ProxyCommand ssh bastion -W %h:%p
```

Comment:
%p need port setting?

Reply:
The two are the same, it just a command line to run

Reply:
1. yeah, `%p` will be 22 if no port is specified.
2. both work!